src/_sass/_base.scss

```diff
@@ -47,10 +47,6 @@ figure {
   }
 }
 
-a {
-  text-decoration: none !important;
-}
-
 p {
```

Comment:
It's not recommended to set a style on all links like this, with an `!important` too. At this point, it was being individually overridden everywhere. Removed.